src/entry-helpers.ts

```diff
@@ -11,7 +11,17 @@ export const genericError = (e: Error) =>
 
 export const waitForCallback = (text: string) => () => {
   const textArea = document.activeElement as HTMLTextAreaElement;
-  if (textArea.value.toUpperCase() !== text.toUpperCase()) {
+  let expectedText = text.toUpperCase();
+  let expectedTextWithoutPeriod = text.replace(/\./g,'').toUpperCase();
+  let actualText = textArea.value.toUpperCase();
```

Comment:
let's strip actual text of periods too. An annoying property I noticed was that periods weren't stripped when they were in a link `[some link](example.com)`. So by only stripping on one side, we will still hang in this case

Reply:
Done.